From c08f49041184b16cc61ef3d2db08c73ba6bbe168 Mon Sep 17 00:00:00 2001
From: Jens Wiklander <jens.wiklander@linaro.org>
Date: Thu, 26 Sep 2024 09:02:29 +0200
Subject: [PATCH 1557/1695] libmbedtls: mbedtls_config_uta.h: enable SHA-384
 and SHA-512 support

Enable SHA-384 and SHA-512 support for user TAs.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 055bc9322d01e22b49ed6c24f2ef505efbf9bba5 libmbedtls: mbedtls_config_uta.h: enable SHA-384 and SHA-512 support]
Change-Id: I26a58b07557169229a9b4569f8394547490d40ca
Signed-off-by: Jens Wiklander <jens.wiklander@linaro.org>
Reviewed-by: Etienne Carriere <etienne.carriere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/437594
Tested-by: Michel JAOUEN <michel.jaouen@st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Michel JAOUEN <michel.jaouen@st.com>
Domain-Review: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 lib/libmbedtls/include/mbedtls_config_uta.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/lib/libmbedtls/include/mbedtls_config_uta.h b/lib/libmbedtls/include/mbedtls_config_uta.h
index 492055676..a21301210 100644
--- a/lib/libmbedtls/include/mbedtls_config_uta.h
+++ b/lib/libmbedtls/include/mbedtls_config_uta.h
@@ -22,6 +22,8 @@
 
 #define MBEDTLS_SHA1_C
 #define MBEDTLS_SHA256_C
+#define MBEDTLS_SHA384_C
+#define MBEDTLS_SHA512_C
 #define MBEDTLS_MD_C
 #define MBEDTLS_MD5_C
 
-- 
2.47.2

